lib/validators/json-schema-legacy.js

```diff
@@ -76,6 +77,10 @@ class JsonSchemaLegacy extends JsonSchemaValidator {
 
     switch (this.jsonSchemaVersion) {
       case 'draftV3':
+        deprecated(
```

Comment:
Somehow the comment on this line got lost. I suggest giving a name that conveys the action of the function, so you don't need to read the function to understand what it does:
 ```suggestion
        logDeprecatedWarning(
```

Reply:
Renamed to `warnOnDeprecation`. Thanks.